src/components/SearchForm.vue

```diff
@@ -12,7 +12,7 @@
         name: 'SearchForm',
         props: [ 'label', 'placeholder', 'query', 'loading', 'onInput', 'onClear'],
         methods: {
-          onSubmit: e => e.preventDefault
+          onSubmit: e => e.preventDefault()
```

Comment:
good catch, and what's the difference?

Reply:
If you don't prevent default, hitting ENTER on the search box will submit the form and reload the page.